book/visual-effects.md

```diff
@@ -578,6 +578,8 @@ complicated to handle in real browsers.
 Surfaces and canvases
 =====================
 
+<a name="surfaces-canvases"></a>
```

Comment:
Every heading in pandoc has an auto-generated name. This one is `#surfaces-and-canvases`. So we don't need to add these anchors to the book.